### PR TITLE
Add a very small quick start and reorganize the front page

### PR DESCRIPTION
**Pull request by satyaog**
Make User Guides and quick start first
Make `mila code` more visible to encourage its usage

### CODE REVIEW

File: docs/Userguide_quick_start.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+recommended to install milatools_ which will help in the :ref:`set up of the
+ssh configuration <mila_init>` needed to securely and easily connect to the
+cluster.
+
+.. _mila_code:
+
+mila code
+---------
+
+milatools_ also helps using and debugging on the Mila cluster. Using the ``mila
```
Comment (lebrice) [inline docs/Userguide_quick_start.rst:16]:
```suggestion
milatools_ also makes it easy to run and debug code on the Mila cluster. Using the ``mila
```

File: docs/Userguide_quick_start.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+code`` command will allow you to use `VSCode <https://code.visualstudio.com/>`_
+on the server. Simply run:
+
+.. code-block:: bash
+
+   mila code path/on/cluster
+
+The details of the command can be found on the `github page of the package
+<https://github.com/mila-iqia/milatools#mila-code>`_. Note that you will first
+need your ssh configuration to be done using the ``mila init`` command. The
```
Comment (lebrice) [inline docs/Userguide_quick_start.rst:26]:
```suggestion
<https://github.com/mila-iqia/milatools#mila-code>`_. Note that you need to first
setup your ssh configuration using ``mila init`` before the ``mila code`` command can be used. The
```

File: docs/index.rst [hunk 3]
```diff
@@ -3,7 +3,8 @@ Mila technical documentation
 
 .. First and only part to the documentation.
 
-Welcome to Mila's technical documentation. See contents below.
+Welcome to Mila's technical documentation. See contents below and have a look
+into the :ref:`short quick start <mila_code>`.
```
Comment (lebrice) [inline docs/index.rst:7]: NOTE: I don't know how the end of this sentence will display (I don't know the .rst format).
```suggestion
Welcome to Mila's technical documentation. If this is your first time here, we recommend you start by checking out the :ref:`short quick start guide <mila_code>`.
```

**Review by lebrice — APPROVED** [review]
:+1:

File: docs/Userguide_quick_start.rst [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+initialisation of the ssh configuration is explained :ref:`here <mila_init>`
+and on the `github page of the package
+<https://github.com/mila-iqia/milatools#mila-init>`_.
+
+.. _milatools: https://github.com/mila-iqia/milatools
```
Comment (lebrice) [inline docs/Userguide_quick_start.rst:31]: (Needs a newline at the end for the linter to be happy.
```suggestion
.. _milatools: https://github.com/mila-iqia/milatools

```

File: docs/index.rst [hunk 5]
```diff
@@ -3,7 +3,9 @@ Mila technical documentation
 
 .. First and only part to the documentation.
 
-Welcome to Mila's technical documentation. See contents below.
+Welcome to Mila's technical documentation. If this is your first time here, we
+recommend you start by checking out the :ref:`short quick start guide
+<mila_code>`.
```
Comment (satyaog) [inline docs/index.rst:8]:
```suggestion
<quick_start>`.
```

**Comment by satyaog** [comment]
Merging, the lint is a bit off in this case

**Pull request merged**